package.json

```
@@ -5,14 +5,15 @@
     "start": "node ./bin/www",
     "test": "NODE_ENV=test node ./node_modules/mocha/bin/mocha ./test/**/*.js --exit",
```

The test script has moved from the sibling directory to the `src` folder.

```suggestion
    "test": "NODE_ENV=test node ./node_modules/mocha/bin/mocha ./src/test/**/*.js --exit",
```